linuxdeploy-plugin-conda.sh

```diff
@@ -9,6 +9,8 @@ fi
 
 script=$(readlink -f "$0")
 
+ARCH=${ARCH:-`uname -m`}
```

Comment:
IIRC `uname -m` is not reliable. But this is just a fallback anyway.

I don't like this "backticks" subshell syntax, and also there's some quoting missing.

```suggestion
ARCH="${ARCH:-"$(uname -m)"}"
```

Reply:
Fixed